.taskfiles/Taskfile_chart.yml

```diff
@@ -0,0 +1,50 @@
+---
+version: '3'
+
+vars:
+  GIT_ROOT:
+    sh: git rev-parse --show-toplevel
+
+tasks:
+
+  # TODO: Use true gotemplate tools
```

Comment:
If it's something that should actually be done, don't forget to raise an issue for it in the repo, otherwise we'll be 100% sure to forget 😄